### DISCUSSION TITLE
How to override hard-coded #daily-notes tag?

### DISCUSSION THREAD

**Discussion by proofer** [epwalsh/obsidian.nvim#583] [Q&A]
I'm new to obsidian.nvim. The first thing I tried was the `ObsidianToday command`. The daily note that it creates has a `daily-notes` tag, that I don't want. (My daily note template, which ls is supplied by `daily_notes[template]` in my config opts, provides a `daily` tag property.)  I also don't need the `id` property. So I thought I would set  
`disable_frontmatter = true`  
but that generated a note with *no* properties, although the body of the note still reflects my template.
Is there a way to create a new daily note with my desired frontmatter as specified in my template?

General observation: hard-coding user data seems like a bad idea. Softener: I sympathize with the difficulty of the task that this plugin undertakes.

**Comment by r2d2rogers** [comment]
Have you tried passing default_tags = { }, in your config for the daily_notes?
